src/interpreter/ByteCodeInterpreter.cpp

```diff
@@ -2149,20 +2149,21 @@ NEVER_INLINE void ByteCodeInterpreter::createFunctionOperation(ExecutionState& s
 
     bool isGenerator = cb->isGenerator();
     bool isAsync = cb->isAsync();
-    Object* proto = state.context()->globalObject()->functionPrototype();
+    Context* functionRealm = code->m_codeBlock->context();
```

Comment:
Minor suggestion.
`code->m_codeBlock` could be replaced by `cb`

Reply:
I changed it! thanks